modern/src/SocketController.js

```diff
@@ -83,7 +74,30 @@ const SocketController = () => {
     return null;
   }, [authenticated]);
 
-  return null;
+  useEffect(() => {
+    setNotifications(events.map((event) => {
+      event.message = `${devices[event.deviceId]?.name}: ${t(prefixString('event', event.type))}`;
+      event.show = true;
+      setTimeout(() => event.show = false, 5000);
+      return event;
+    }));
+  }, [events]);
```

Comment:
This uses devices, but we don't want this to be triggered every time devices update because devices are updated frequently. If we add devices to the array the snackbars will be displayed every time new devices arrive even a new event wasn't arrived.

Reply:
No, wait.. you need to clear notifications once they are dismissed. Looks like that's another thing that's missing.

Reply:
That's right. I'll do that.